templates/home/index.html.twig

```diff
@@ -1,7 +1,64 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}HomeWork...{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+    <li class="list-group-item list-group-item-info">Studentai</li>
+    {% for student in someVariable %}
+
+        {% for name in student.students%}
+            <li class="list-group-item">
+                <a href="{{path("studentas")}}?student={{name|url_encode}}&project={{student.name|url_encode}}">{{name}}</a>
+                (
+                <span class="badge">Mentorius</span> 
+                {% for name in student.mentors %}
+                {{name}}
+                {% endfor %}
+                )
+            </li>
+        {% endfor %}
+    {% endfor %}
+   </ul>
+
+    <div class="m-4">
+        <form action="/studentas?student="student"&project="project" method="get">
```

Comment:
Naudok `path` komandą. O ne `hardcoded` reikšmes.

Gi turi apsirašęs `@Route("/studentas", name="studentas")`

Reply:
Ir pavizdyje adresas yra be argumentų:
![image](https://user-images.githubusercontent.com/1453957/69915915-18def380-145d-11ea-8299-61c692c749be.png)

Mokykis atidumo – dideliuose projektuose `padaryti belekaip` reikš daug daugiau laiko ieškant klaidos